html/pages/device/edit/device.inc.php

```diff
@@ -40,6 +40,25 @@
         } else {
             $update_message = "Device record update error.";
         }
+
+        if ($_POST['hostname'] !== '' && $_POST['hostname'] !== $device['hostname']) {
+            if ($_SESSION['userlevel'] === 10) {
```

Comment:
We have functions to check admin levels, something like is_admin(), I can't remember 100%